vignettes/EIRprevmatch.Rmd

```diff
@@ -72,7 +63,6 @@ malEq_P2E <- cbind.data.frame(EIR = malEq_EIR, prev = malEq_prev)
 
 # View the dataframe containing the EIR and matching PfPR2-10 values:
 head(malEq_P2E, n = 7)
-
 ```
 
 As this method does not involve running simulations it can return matching *Pf*PR~2-10~ for a wide range of EIR values very quickly. However, it is only viable for systems at steady state, with a fixed set of parameter values, and only enables the user to capture the effects of the clinical treatment of cases. Often when running `malariasimulation` simulations, one or both of these conditions are not met. In this example, we have set the proportion of cases effectively treated (`ft`) to 0.45 to capture the effect of clinical treatment with antimalarial drugs. However, the user may, for example, also wish to include the effects of a broader suite of interventions (e.g. bed nets, vaccines, etc.), or to capture changes in the proportion of people clinically treated over time. In these cases, a different solution would therefore be required.
```

Comment:
In the section entitled "*Pf*PR~2-10~ matching using malariasimulation" there are three instances where PfPR2-10 needs to be converted to *Pf*PR~2-10~

Reply:
This hasn't shown up the way I'd hoped. To be clearer, there are three times where PfPR2-10 is written without using subscripting